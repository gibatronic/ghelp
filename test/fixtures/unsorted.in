# task a
a:
  echo task a

# task c
c:
  echo task c

# task b
b:
  echo task b
