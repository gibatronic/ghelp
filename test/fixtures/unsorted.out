a  task a
b  task b
c  task c
